README.md

```diff
@@ -2,51 +2,69 @@
 
 This repository contains tests.
 
-The infra for the tests can be found in https://github.com/RedHatQE/openshift-python-wrapper and https://github.com/RedHatQE/openshift-python-utilities
-flake8 plugins defined in .flake8 can be found in https://github.com/RedHatQE/flake8-plugins
+The infra for the tests can be found in <https://github.com/RedHatQE/openshift-python-wrapper> and <https://github.com/RedHatQE/openshift-python-utilities>
+flake8 plugins defined in .flake8 can be found in <https://github.com/RedHatQE/flake8-plugins>
+
+## How to run
+
+### Setup VirtualEnv
 
-## Setup VirtualEnv
 Use [poetry](https://python-poetry.org/docs/) to manage virtualenv.
+
 ```bash
 pip install poetry
 ```
+
 After installation, run:
+
 ```bash
 poetry install
 ```
+
 To get current env info
+
 ```bash
 poetry env info
 ```
+
 To get poetry virtualenv names
+
 ```bash
 poetry env list
 ```
+
 To remove current env
+
 ```bash
 poetry env remove <env name>
 ```
+
 To clear poetry cache (needed before update if there is an existing update but the package is not updated)
+
 ```bash
 poetry cache list # get poetry available cache list
 poetry cache clear --all <cache name> # delete all cache (except _default_cache)
 ```
+
 To update one package
+
 ```bash
 poetry update openshift-python-wrapper
 ```
 
-# Getting started
-
-## Prepare a cluster
+### Prepare a cluster
 
 This project runs tests against an OCP cluster running on OSD / ROSA.
 Export the cluster's kubeconfig file as KUBECONFIG
+
 ```bash
-ocm get clusters -p search="name like '"<cluster name>"%'" | jq -r  '.items | .[] | .id' | xargs -I {} ocm get /api/clusters_mgmt/v1/clusters/{}/credentials | jq -r .kubeconfig > ~/kubeconfig ; export KUBECONFIG=~/kubeconfig
+ocm get clusters -p search="name like '"<cluster name>"%'" | jq -r  '.items | .[] | .id' \
+| xargs -I {} ocm get /api/clusters_mgmt/v1/clusters/{}/credentials | jq -r .kubeconfig > \
+~/kubeconfig ; export KUBECONFIG=~/kubeconfig
```

Comment:
```suggestion
export CLUSTER_NAME=<cluster_name>

ocm get clusters -p search="name like '"$CLUSTER_NAME"%'" | jq -r  '.items | .[] | .id' \
| xargs -I {} ocm get /api/clusters_mgmt/v1/clusters/{}/credentials | jq -r .kubeconfig > \
~/kubeconfig ; export KUBECONFIG=~/$CLUSTER_NAME-kubeconfig
```

Reply:
This is a matter of preference; I for once prefer not to set `CLUSTER_NAME`